controllers/dashboard_controller.go

```diff
@@ -410,6 +413,7 @@ func (r *GrafanaDashboardReconciler) fetchDashboardJson(dashboard *v1beta1.Grafa
 
 func (r *GrafanaDashboardReconciler) UpdateStatus(ctx context.Context, cr *v1beta1.GrafanaDashboard) error {
 	cr.Status.Hash = cr.Hash()
+	cr.Status.NoMatchingInstances = false
```

Comment:
Could you, please, explain why this line is added here and not inside a function that checks for presence of matching instances? (assuming `UpdateStatus` can be called from anywhere)
Btw, I see that in different controllers it is set to different values.

Reply:
I have moved in to a separate function witch handles that logic. 
Please have a look now